sample/library/src/test/kotlin/app/cash/paraphrase/sample/library/JvmDateTimeConverter.kt

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (C) 2023 Cash App
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.paraphrase.sample.library
+
+import app.cash.paraphrase.DateTimeConverter
+import com.ibm.icu.util.Calendar
+import com.ibm.icu.util.TimeZone
+import com.ibm.icu.util.ULocale
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.LocalTime
+import java.time.OffsetDateTime
+import java.time.OffsetTime
+import java.time.ZoneOffset
+import java.time.ZonedDateTime
+
+/**
+ * Converts `java.time` types used by Paraphrase to a [Calendar] that can be used by ICU to format.
+ */
+// TODO: Ship this in a new artifact?
```

Comment:
Probably needs a new `runtime-test` artifact, but I didn't want to bother unless we decide to adopt this solution.

Reply:
Are we able to also run the instrumentation tests with a parameterized runner that exercises both types? That way we ensure behavior always remains the same.

Reply:
I can try. Not actually sure what will happen if I try to run the IBM package on an Android device – will it find the necessary formatting rules or will they be missing? I'll find out!

Reply:
Hmm true. I'm not sure from where it pulls them. I would assume they're embedded in the jar.

Reply:
Seems to work, except ICU4J uses a `NNBSP` between before "AM" and "PM" while Android uses a normal space. Annoying but workable.